CONTRIBUTING.md

```diff
@@ -95,6 +95,32 @@ Run following command from `website` folder. It will build the docs and serve th
 ./scripts/build_website.sh
 ```
 
+You can also perform spell checks on documentation automatically (besides IDEs) using [```sphinxcontrib-spelling```](https://sphinxcontrib-spelling.readthedocs.io/en/latest/install.html)
```

Comment:
Would it be possible to add spell check as an automated linter Eg: https://github.com/marketplace/actions/github-spellcheck-action ?
^ This uses PySpelling and I am not sure if it is compatible with all the files we have.